Makefile

```diff
@@ -1,25 +1,29 @@
 DOCKER ?= docker
+CONTAINER ?= container
 # Installation directories.
 PREFIX ?= $(DESTDIR)/usr
 BINDIR ?= $(PREFIX)/bin
 MANDIR ?= $(PREFIX)/share/man
 UNITDIR ?= $(PREFIX)/lib/systemd/system
-DSSLIBDIR ?= $(PREFIX)/lib/docker-storage-setup
+CSSLIBDIR ?= $(PREFIX)/lib/container-storage-setup
 SYSCONFDIR ?= $(DESTDIR)/etc/sysconfig
 
 .PHONY: clean
 clean:
 	-rm -rf *~ \#* .#*
 
 .PHONY: install
-install:
-
-	install -D -m 755 ${DOCKER}-storage-setup.sh ${BINDIR}/${DOCKER}-storage-setup
+install-docker:
 	install -D -m 644 ${DOCKER}-storage-setup.service ${UNITDIR}/${DOCKER}-storage-setup.service
```

Comment:
I mean "make install" will override docker.service already installed on the system? That does not sound right, does it? "make install" should not get into business of installing systemd unit files I think and it should be left to rpm packaging?